dotCMS/src/main/java/com/dotmarketing/viewtools/content/ContentTool.java

```diff
@@ -76,21 +75,14 @@ public void init(Object initData) {
 		}
 		
 		tmDate=null;
-		ADMIN_MODE=false;
-		PREVIEW_MODE=false;
-		EDIT_MODE=false;
+
 		HttpSession session = req.getSession(false);
-		
-		
+		PageMode mode = PageMode.get(req);
+		EDIT_OR_PREVIEW_MODE=!mode.showLive;
 		if(session!=null){
 			tmDate = (String) session.getAttribute("tm_date");
 			boolean tm=tmDate!=null;
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "tm". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "tm" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)